Iks 1.3 docs (#2660)
kubeflow/website: content/en/docs/distributions/ibm/deploy/deployment-process.md

## Kubeflow installation

> As of Kubeflow 1.3 the official installation documentation uses a combination of `kustomize` and `kubectl` to install Kubeflow. Using `kfdef` and `kfctl` will also continue to be a way to install Kubeflow. 

Reviewer: Suggestion:

```suggestion
As of Kubeflow 1.3, the official installation documentation uses a combination of `kustomize` and `kubectl` to install Kubeflow. Using `kfdef` and `kfctl` will also continue to be a way to install Kubeflow. 
```

      ```shell
      export PATH=$PATH:<path to where kfctl was unpacked>
      ```

> ### Install kubectl and kustomize

Author: @Tomcli

* [Install kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) 
* [Download kustomize 3.2.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0)

> To use the `kustomize` binary you would need make it executable and move it to your path.
> 
> To add kustomize in your global path you can do the following

Reviewer: Suggestion:

- "To do X, [you] do Y..."

```suggestion
To use the `kustomize` binary, you need to make it executable and move it to your path.

To add kustomize to your global path, run the following commands:
```

```bash
wget https://github.com/kubernetes-sigs/kustomize/releases/download/v3.2.0/<distribution>
chmod +x <distribution>
mv <distribution> /usr/local/bin/kustomize
```

> Your machine might already have `kustomize` installed. If you want to just add this version of `kustomize` to your path temporarily you can do it this way

Reviewer: Suggestion:

```suggestion
Your machine might already have `kustomize` installed. If you want to temporarily add this version of `kustomize` to your path, run the following commands:
```
